socorro/cron/crontabber_app.py

```diff
@@ -105,11 +101,9 @@ def jobs_converter(path_or_jobs):
     else:
         input_str = path_or_jobs
 
+    # At this point, we have a big string of jobs
     from_string_converter = classes_in_namespaces_converter_with_compression(
-        reference_namespace=Namespace(),
-        list_splitter_fn=line_splitter,
-        class_extractor=pipe_splitter,
-        extra_extractor=get_extra_as_options
```

Comment:
I folded these into the arg defaults and nixed the other defaults because we never use them anywhere.